assets/js/modules/tagmanager/hooks/useExistingTagEffect.js

```diff
@@ -42,5 +42,13 @@ export default function useExistingTagEffect() {
 				await selectContainerByID( existingTag );
 			}
 		} )();
-	}, [ hasExistingTag, existingTag, hasExistingTagPermission, existingTagPermission ] );
+	},
+	[
+		hasExistingTag,
+		existingTag,
+		hasExistingTagPermission,
+		existingTagPermission,
+		selectAccount,
+		selectContainerByID,
+	]	);
```

Comment:
Surprised ESLint doesn't care about this, but how about:

```suggestion
	}, [
		hasExistingTag,
		existingTag,
		hasExistingTagPermission,
		existingTagPermission,
		selectAccount,
		selectContainerByID,
	] );
```